src/app/shared/theme.service.ts

```diff
@@ -148,6 +150,31 @@ const defaultTheme: Theme = {
   },
 };
 
+const mapKviToTypographyLevel = (
+  items: KeyValueIdentifier[],
+  key: string,
+  defaultValue: ThemeMaterialTypographyLevel
+) => {
+  const value = getKviValue(items, key, '');
+  if (value === '') {
+    return defaultValue;
+  }
+
+  const parts = value.split(', ');
```

Comment:
Since these values will come from typed user inputs, I think we need at least to be able to handle spacing variations like "size,height,weight" or " size , height  , weight "

Reply:
Yes, you're correct. I've updated this parsing to remove all whitespace before splitting on commas, and updated the tests to catch both possible cases.